SauceDemo.Business/Steps/LoginPageSteps.cs

```diff
@@ -0,0 +1,45 @@
+﻿using OpenQA.Selenium;
+using SauceDemo.Business.Pages;
+
+namespace SauceDemo.Business.Steps
+{
```

Comment:
I am not sure if you require this steps :) everything could be located in the test class and it would look great, I promise :)